frontend/src/Editor/Components/Table/columns/actions.jsx

```diff
@@ -56,6 +65,15 @@ const generateActionsData = ({ actions, columnSizes, defaultColumn, fireEvent, s
               return rightActions().map((action) => (
                 <button
                   key={action.name}
+                  disabled={
+                    action?.buttonState !== undefined
+                      ? typeof resolveReferences(action.buttonState, currentState) == 'string'
```

Comment:
Here also, the same data is resolved over and over again several times. Please only resolve it once.